src/services/zendesk/filterTicketsByKeyword.ts

```diff
@@ -0,0 +1,50 @@
+import getNewTickets from "./getNewTicketsService";
+import ticket from "../../models/ticketModel";
+require('dotenv').config();
+
+function checker(subject:string,keywords:string[]) {
+    
+    for (let i:number = 0; i < keywords.length; i++) {
+      if (subject.indexOf(keywords[i]) > -1) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+const filterTicketsByKeyword = async () => {
+    let newTickets:ticket[] = await getNewTickets();
+    let keywords1 = ['access','ftp','sds', 'pds'];
+    let keywords2 = ['st','sql','css','js'];
```

Comment:
Keywords should be imported from a separate file - not managed here (for easier maintenance in the future, if we would like to refer to the keywords in other places like frontend app etc). 
In /src folder you can create CONSTANTS.ts file and put two named exports there, each for L1 and L2 tickets (we will use this file for any constant values in the future)